lib/Doctrine/Common/Collections/Collection.php

```diff
@@ -283,7 +275,7 @@ public function partition(Closure $p): array;
      * @return int|string|bool The key/index of the element or FALSE if the element was not found.
      * @psalm-return TKey|false
      */
-    public function indexOf($element);
+    public function indexOf(mixed $element): int|string|bool;
```

Comment:
```suggestion
    public function indexOf(mixed $element): int|string|false;
```

We never return `true`, do we?